src/components/Skeleton/Skeleton.jsx

```diff
@@ -14,7 +14,7 @@ import { BEMClass } from "../../helpers/bem-helper";
 const SKELETON_CSS_BASE_CLASS = "monday-style-skeleton";
 const bemHelper = BEMClass(SKELETON_CSS_BASE_CLASS);
 
-export const Skeleton = ({ type, size, className, width, height }) => {
+const Skeleton = ({ type, size, className, width, height }) => {
```

Comment:
fix import (for match other imports in this project and for the index export to work well)